katdal/datasources.py

```diff
@@ -380,11 +380,10 @@ def __init__(self, telstate, capture_block_id, stream_name,
         if chunk_store is None:
             data = None
         else:
-            if telstate.get('need_weights_power_scale', False):
-                corrprods = telstate['bls_ordering']
-            else:
-                corrprods = None
```

Comment:
I think it's fine, but just to check: is it possible that bls_ordering might be missing in the old observations that lacked need_weights_power_scale?

Reply:
I just found the oldest RDB in the archive: 1524571621 captured on 2018-04-24 (holography, of course 😆). That has `bls_ordering` so I think it's OK. For the record, `need_weights_power_scale` appeared on site on 2018-10-17.